docker/quick-setup/systemProxy/README.md

```diff
@@ -0,0 +1,21 @@
+# MongoDB
+
+Here is a docker-compose to run APIM with MongoDB as database and using [mitmproxy](https://mitmproxy.org/).
+
+---
+> For more information, please read :
+> https://documentation.gravitee.io/apim/getting-started/configuration/repositories/mongodb
+---
+
+## How to run ?
+
+⚠️ You need a license file to be able to run Enterprise Edition of APIM. Do not forget to add your license file into `./.license`.
+
+`APIM_VERSION={APIM_VERSION} docker-compose up -d ` 
```

Comment:
I would say we don't need it

Reply:
But what if you need system proxy with an entreprise feature ? In the future 😉